Assets/Scripts/Enemy/Minions/BubbleController.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BubbleController : FollowBossController
+{
+    protected int _bubbleHealth = 20;
+    private Vector3 _startPosition, _endPosition;
+    private float _desiredDuration = 1.2f;
+    private float _elapsedTime, _percentageComplete;
+    [SerializeField] private AnimationCurve _curve;
+
+    // Start is called before the first frame update
+    protected override void Start()
+    {
+        base.Start();
+        _enablePause = true;
+        GetComponent<BossHealth>().SetMaxHealth(_bubbleHealth);
+        _startPosition = transform.position;
+        _endPosition = _player.position;
+    }
+
+    protected override void FixedUpdate()
+    {
+        _elapsedTime += Time.deltaTime;
+        _percentageComplete = _elapsedTime / _desiredDuration;
+        transform.position = Vector3.Lerp(_startPosition, _endPosition, _curve.Evaluate(_percentageComplete));
```

Comment:
Maybe this logic should be in Move, rather than FixedUpdate, since FixedUpdate is calling Move already.

Reply:
When i move this to Move the bubbles stop moving. Any idea why it's not equivalent?